runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -600,21 +606,15 @@ TR_RelocationRecord::computeHelperAddress(TR_RelocationRuntime *reloRuntime, TR_
 #undef FLAGS_RELOCATION_FLAG_MASK
 
 
-bool
+TR_RelocationRecordIgnoreCode
 TR_RelocationRecord::ignore(TR_RelocationRuntime *reloRuntime)
```

Comment:
I don't like this being called `ignore()` anymore, given how you're changing it. You're asking more a question like "what should I do with this relocation record?" .  Maybe `action()` ? rather than `ignore()`? then the current set of reasonable answers can be "ignore it", "apply relocation", "fail compilation", or any other variant that makes sense (as opposed to "internal error")...

You're not really asking if you can ignore it, you're asking more a question about what action to perform for this relocation: ignore it, perform it, or "internal error". I feel that would be a bit more clear than asking `ignore()` and is

Reply:
Heh naming is hard; I was trying to think of what to change `ignore()` to, but I just wasn't creative enough :P. I'll make the requested change.